lib/primitives/block.js

```diff
@@ -842,14 +843,37 @@ class Block extends AbstractBlock {
     return obj instanceof Block;
   }
 
-  /*
+  /**
+   * Get parameter values for a filterType
+   * @param {*} filterType
+   * @returns {@Object}
+   */
+
+  buildFilterParams(filterType) {
+    const params={P: 19,
+                M: 784931};
```

Comment:
style here doesnt look quite right either, I'd either go with

```js
const params={
  P: 19,
  M: 784931
};
```

or

```js
const params = {P: 19, M: 784931};
```

But actually given the switch construciton Im not sure initializing with these parameters is best idea anyway